resources/ottawa.csv

```diff
@@ -1,7 +1,7 @@
+,win_x86,win_x86-64_nocmprssptrs,win_x86-64_cmprssptrs,win_x86-64,linux_x86,linux_x86-64_nocmprssptrs,linux_x86-64_cmprssptrs,linux_x86-64,linux_ppc,linux_ppc-64_nocmprssptrs,linux_ppc-64_cmprssptrs,linux_ppc-64_nocmprssptrs_le,linux_ppc-64_cmprssptrs_le,linux_ppc-64_le,linux_390,linux_390-64_nocmprssptrs,linux_390-64_cmprssptrs,linux_390-64_cmprssptrs_zt,linux_390-64,linux_390-64_zt,aix_ppc,aix_ppc-64_nocmprssptrs,aix_ppc-64_cmprssptrs,aix_ppc-64,zos_390,zos_390_zt,zos_390-64_nocmprssptrs,zos_390-64_cmprssptrs,zos_390-64_cmprssptrs_zt,zos_390-64,zos_390-64_zt,linux_arm,osx_x86-64_nocmprssptrs,osx_x86-64_cmprssptrs,osx_x86-64,osx_aarch64_nocmprssptrs,bsd_x86-64,sunos_sparcv9-64,sunos_x86-64,linux_aarch64_nocmprssptrs,linux_aarch64_cmprssptrs,linux_aarch64,linux_riscv64_nocmprssptrs,linux_riscv64_cmprssptrs,linux_riscv64,linux_loongarch64_nocmprssptrs,linux_loongarch64_cmprssptrs,linux_loongarch64,alpine-linux_x86-64,alpine-linux_aarch64,default-64,default-none64
+plat,win_x86-32,win_x86-64_ncr,win_x86-64_cr,win_x86-64,linux_x86-32,linux_x86-64_ncr,linux_x86-64_cr,linux_x86-64,linux_ppc-32,linux_ppc-64_ncr,linux_ppc-64_cr,linux_ppcle-64_ncr,linux_ppcle-64_cr,linux_ppc-64,linux_390-31,linux_390-64_ncr,linux_390-64_cr,linux_390-64_cr_zt,linux_390-64,linux_390-64_zt,aix_ppc-32,aix_ppc-64_ncr,aix_ppc-64_cr,aix_ppc-64,zos_390-31,zos_390-31_zt,zos_390-64_ncr,zos_390-64_cr,zos_390-64_cr_zt,zos_390-64,zos_390-64_zt,linux_arm,osx_x86-64_ncr,osx_x86-64_cr,osx_x86-64,osx_aarch64_ncr,bsd_x86-64,sunos_sparcv9-64,sunos_x86-64,linux_aarch64_ncr,linux_aarch64_cr,linux_aarch64,linux_riscv64_ncr,linux_riscv64_cr,linux_riscv64,linux_loongarch64_ncr,linux_loongarch64_cr,linux_loongarch64,alpine-linux_x86-64,alpine-linux_aarch64,default-64,default-none64

```

Comment:
To add platforms into Ottawa.csv, we need to fill out yes/no for each row of variation. I think the platform you added here is standard in terms of variation, so we can remove the change in Ottawa.csv file and rely on the default behaviour. @llxia any comment?

Reply:
I agree. There is no need to update ottawa.csv in this case.